level-4/fundamentals-of-computer-science/student-notes/fabio-lama/src/propositional_logic.adoc

```diff
@@ -0,0 +1,38 @@
+= Cheatsheet - Propositional Logic
+Fabio Lama <fabio.lama@pm.me>
+:description: Module: CM1025 Fundamentals to Computer Science, started 25. October 2022
+:doctype: article
+:sectnums: 4
+:stem:
+
```

Comment:
i had my royalty rights used by someone that has been hacking into my system. it has created more headaches than i wanted to notice for everyone that has been watching it go on. i feel as an O.S. system with licensing for communication and internet servicing and Networking. I have fifteen people plus some riding on my system that i cant get out. they have stole more money from me than what they are aware of me being able to watch my buffer times about. i do need a little security while i keep my investigation on going...

Reply:
Uhm... what?